R/pkg/DESCRIPTION

```diff
@@ -59,3 +59,4 @@ Collate:
     'window.R'
 RoxygenNote: 5.0.1
 VignetteBuilder: knitr
+SystemRequirements: Apache Spark
```

Comment:
I'm not sure we should add this - my experience with SystemRequirements is that there is a fix list of things that it will understand, and I'm reasonably sure "Apache Spark" is not one of them, and likely we cause build or submission failure by having it.

Reply:
and FYI, this was the relevant part from the email
```
Apparently you expect some installed Hadoop or SPark software for running the vignettes?
But there is no SystemRequeirements field....?
```

Reply:
And btw, the issue with vigenettes is already fixed.